### PR TITLE
Add Media Object migration to R4

### PR DESCRIPTION
**Pull request by bkeese**

### CODE REVIEW

File: db/migrate/20150824155143_r3_media_object_to_r4.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class R3MediaObjectToR4 < ActiveRecord::Migration
+  def up
+    say_with_time("R3->R4") do
+      prefix = Avalon::Configuration.lookup('fedora.namespace')
+      ActiveFedora::Base.reindex_everything("pid~#{prefix}:*")
```
Comment (cjcolvar) [inline db/migrate/20150824155143_r3_media_object_to_r4.rb]: Is this reindex really necessary?  What purpose does it serve?

Comment (bkeese) [inline db/migrate/20150824155143_r3_media_object_to_r4.rb]: Because identifiers are supposed to be searchable.

Comment (bkeese) [inline db/migrate/20150824155143_r3_media_object_to_r4.rb]: I'm not sure. I'll guess I was just following the example of a previous migration.

**Comment by cjcolvar** [comment]
:+1:

**Pull request merged**